### ISSUE TITLE
v3: PrismicImageFluidType  is not compatible with gatsby-image

### ISSUE THREAD

**Issue by rubas** [angeloashmore/gatsby-source-prismic#174]
Gatsby-image's `FluidObject` | `FixedObject` expects `sizes` to be a string but I always get back `null`.

```
"data": {
  "main_image": {
    "fluid": {
      "sizes": null
     }
  }
}
```

## gatsby-image
```
export interface FluidObject {
  aspectRatio: number
  src: string
  srcSet: string
  sizes: string
  base64?: string
  tracedSVG?: string
  srcWebp?: string
  srcSetWebp?: string
  media?: string
}
```

**Comment by giacomoalonzi** [comment]
@rubas do you get "Invalid prop `fluid` supplied to `Image`" in console?

**Comment by thejoshsmith** [comment]
@giacomoalonzi I'm getting that exact error.

**Comment by rubas** [comment]
Exactly. Instead of `null` it should be `""`.

@thejoshsmith 
My work around is to add an empty string.
`<Img fluid={{ ...fluid, sizes: "" } as FluidObject} alt={alt ?? undefined} />`

**Comment by pbrandone** [comment]
here's the problem: https://github.com/angeloashmore/gatsby-source-prismic/blob/v3-beta/src/common/getGatsbyImage.js#L169

**Comment by angeloashmore** [comment]
Hi everyone, this should be fixed in `v3.0.0-beta.24`:

https://github.com/angeloashmore/gatsby-source-prismic/blob/6cc5888811ac1e42215827ee5a96a719855551a2/src/gatsbyImage.ts#L164-L172

**Comment by steverandy** [comment]
@angeloashmore not having sizes value have negative impact in image loading.
I think gatsby-image limits the size to the maxWidth set in the query.
